src/LudoGame/GameEngine/GameFlow.cs

```diff
@@ -0,0 +1,304 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Drawing;
+
+namespace GameEngine
+{
+
+    //CRITCAL BUG:::::: Loop does not display die rolls
```

Comment:
Nice to know (Hint): Visual Studio have a task list function and in the is it possible to add your own tasks types, so you could add the task type BUG (to your own visual studio instance) and then would you be able to write:
BUG: CRITICAL loop does not display die rolls
And the bug would show up in your task list